constructor/pkg/construct.yaml

```diff
@@ -21,8 +21,11 @@ specs:
   - jupyter-jaeger
   - pandas <0.26
   - pymapd
+  - pyarrow 0.15.0
   - python >=3.7
   - ibis-framework
   - jupyterlab >=2.2
   # needed for pkg building
   - conda
+
+post_install: ../post-install.sh  # [osx]
```

Comment:
IIRC correctly post_install scripts are not run for the pkg installer due to a bug in constructor.  You should check this.

Reply:
@scopatz as pkg is not priority for now, could I work on that as a follow-up task?
locally the building is taking a lot of time ... 
I can remove the package pkg from the release if it is not working (the priority right now is the sh)
I can also check (for a follow-up task) if I can check the pkg inside the CI

how does it sound?

Reply:
Yeah that's fine